app/subscriber/src/components/search-with-logout/SearchWithLogout.tsx

```diff
@@ -1,32 +1,43 @@
 import React from 'react';
 import { BiLogOut } from 'react-icons/bi';
-import { FaUserCircle } from 'react-icons/fa';
-import { useContent } from 'store/hooks';
-import { LogicalOperator, Show, Text, useKeycloakWrapper, useWindowSize } from 'tno-core';
+import { FaSearch, FaUserCircle } from 'react-icons/fa';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Button, Row, Show, Text, useKeycloakWrapper, useWindowSize } from 'tno-core';
 
 import * as styled from './styled';
 
 /** The component containing the search bar and the logout button for the subscriber application. Responsive and adjusts to the viewing device*/
 export const SearchWithLogout: React.FC = () => {
   const keycloak = useKeycloakWrapper();
-  const [, { storeFilterAdvanced }] = useContent();
+  const [searchItem, setSearchItem] = React.useState<string>('');
+  const navigate = useNavigate();
   const { width } = useWindowSize();
+  const { id } = useParams();
```

Comment:
Where is the `id` parameter coming from?  The path you have below is `/search/${searchItem}`.